src/Containers/DenseAxisArray.jl

```diff
@@ -148,17 +148,26 @@ function Base.get(
     return get(x.data, key[1] => key[2], default)
 end
 
-_abstract_vector(x::AbstractVector) = x
+function _abstract_vector(x::AbstractVector)
```

Comment:
Can't you do that with dispatch `_abstract_vector(::AbstractVector{<:CartesianIndex}) = error...`